src/coins.cpp

```diff
@@ -1557,6 +1576,65 @@ CValidationState::Code CCoinsViewCache::IsScTxApplicableToState(const CTransacti
     return CValidationState::Code::OK;
 }
 
+#ifdef ENABLE_ADDRESS_INDEXING
```

Comment:
move ifdef to line 1605 and 1632 as these methods must also manage txIndex update on sidechain ceasing